src/dev/build/lib/scan_copy.test.ts

```diff
@@ -79,7 +79,7 @@ it('rejects if neither path is absolute', async () => {
   );
 });
 
-it('copies files and directories from source to dest, including dot files, creating dest if necessary, respecting mode', async () => {
+it.skip('copies files and directories from source to dest, including dot files, creating dest if necessary, respecting mode', async () => {
```

Comment:
Hi What is the error message for this one here. Does not seem like a x-pack related error

Reply:
@zhyuanqi this is a flaky test identified in PR: https://github.com/Tengda-He/kibana/pull/28. It expects "Is_windows" to be either 644 or 666. This works on local machine and outputs 644. However, the output is flaky on jenkins and results "664". This test file has also been removed in branch 7.10